.spi.yml

```diff
@@ -5,9 +5,8 @@ builder:
     scheme: ComposableArchitecture
   - platform: macos-xcodebuild
     scheme: ComposableArchitecture
-  - platform: macos-xcodebuild-arm
-    scheme: ComposableArchitecture
```

Comment:
Unrelated but while I was at it I removed this - we're running all builds on ARM now and the `macos-xcodebuild-arm` platform variant doesn't exist anymore.